.gitignore

```diff
@@ -1,4 +1,5 @@
 .docusaurus/
+.notebook/
```

Comment:
IMAO, as long as this is part of the project, it should not be added here. 
Files listed in .gitignore are usually temporary files, sensitive data, or local files.

Reply:
You might be right about it being useful, but that should not be a part of this PR.

Reply:
I will update it